cas/incident-reporting.md

```diff
@@ -0,0 +1,101 @@
+# Incidents
+
+Incidents can happen. Things do not always go as planned, and that can be okay. However, when incidents occur, the underlying issues (i.e. root causes) must be identified and remediated to discourage the incident from occurring again. Formally documenting the incident in a report encourages an understanding of all contributing root causes, and it presents the opportunity to clearly communicate a remediation plan to reduce the probability of recurrence. 
+
+## Objectives and Expectations
+
+The purpose of Incident Reporting is to help us work together to build a more secure web. 
+
+Depending on the root programs in which a CA Owner participates, it may be required to:
+* create [Audit Incident Reports](audit-incident-report) when audits have non-conformities, qualifications, or modified opinions; and/or
+* create [Incident Reports](incident-report) for all other incidents.
+
+These reports provide lessons learned and transparency about the steps the CA Owner takes to address the immediate issue and prevent future issues. If the underlying problem goes unfixed, then other issues that share the same root cause will subsequently surface. Additionally, Incident Reports help the Web PKI ecosystem as a whole because they promote continuous improvement, information sharing, and highlight opportunities to define and adopt improved practices, policies, and controls.
+
+CA Incident Reports are expected to:
```

Comment:
I know it would be a bunch of work, but I think positive and negative examples of these attributes would be very helpful. (As an example, I'm not sure what you mean to indicate differently between "candid" and "transparent" in the context of a CAIR.)

Reply:
#187 attempts to better clarify use of terms "candid", "transparent", and "timely."